Assets/Scripts/Enemy/EnemyMovement.cs

```diff
@@ -4,15 +4,61 @@
 
 public class EnemyMovement : MonoBehaviour
 {
+    private GameObject player;
+    private Rigidbody2D myBody;
+    Vector2 movement;
+
+    [SerializeField] float moveSpeed = 1.0f;
+    [SerializeField] float attackDistance = 2.0f; //should be 0 or very small for melee enemies
+    public float aggroRange;
+    bool isAggro = false;
+
     // Start is called before the first frame update
     void Start()
     {
-        
+        isAggro = false;
+        if (player == null)
+        {
+            player = GameObject.FindGameObjectWithTag("Player");
```

Comment:
remove the nullcheck, Starts only called once so this will never run more than once.